debian/postinst

```diff
@@ -101,6 +82,13 @@ jicofo {
         # Make sure the client-proxy is set correctly.
         hocon -f $HOCON_CONFIG set jicofo.xmpp.client.client-proxy "focus.$JVB_HOSTNAME"
 
+        # Set the client XMPP config in jicofo.conf. The values were either read from an existing
+        # /etc/jitsi/jicofo/config or generated above.
+        hocon -f $HOCON_CONFIG set "jicofo.xmpp.client.xmpp-domain" "$JICOFO_HOSTNAME"
```

Comment:
If you want to migrate config, this will not work. This `if` is when there was no old config or we are reconfiguring jicofo because the domain changed. 
And if you drop the handling in the code, those that are updating will stop working for them.

Reply:
It's actually outside the `if`